src/token_request_handler.ts

```diff
@@ -46,6 +51,24 @@ export class BaseTokenRequestHandler implements TokenRequestHandler {
     return (response as TokenErrorJson).error === undefined;
   }
 
+  performRevokeTokenRequest(
+      configuration: AuthorizationServiceConfiguration,
+      request: RevokeTokenRequest): Promise<boolean> {
+    let revokeTokenResponse = this.requestor.xhr<any>({
+      url: configuration.revocationEndpoint + '?' + this.utils.stringify(request.toStringMap()),
+      dataType: 'json',  // adding implicit dataType
+      method: 'GET'
```

Comment:
The fact is that Google's implementation appears to require GET on the revocation endpoint, instead of POST https://developers.google.com/identity/protocols/OAuth2WebServer#tokenrevoke. The client authentication is also an open question. In order to work with most spec-conformant providers, I think this library should support client authentication.